install/overlays/azure/kustomization.yaml

```diff
@@ -38,7 +38,7 @@ secretGenerator:
 - name: ssh-key-secret
   namespace: confidential-containers-system
   files: # key generation example: ssh-keygen -f ./id_rsa -N "" && sudo cat id_rsa.pub >> /root/.ssh/authorized_keys
```

Comment:
also this comment is not needed as it used to authorize the public key on the libvirt machine

Reply:
The part after `&&` unnecessary!